code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm

```diff
@@ -129,21 +148,23 @@
 	item_flags = NOBLUDGEON
 	w_class = WEIGHT_CLASS_NORMAL
 	layer = MOB_LAYER
+	///A list of all the mob types unable to use this item.
 	var/list/banned_mobs
 
 /obj/item/fugu_gland/afterattack(atom/target, mob/user, proximity_flag)
 	. = ..()
-	if(proximity_flag && isanimal(target))
-		var/mob/living/simple_animal/A = target
-		if(A.buffed || (A.type in banned_mobs) || A.stat)
-			to_chat(user, "<span class='warning'>Something's interfering with [src]'s effects. It's no use.</span>")
-			return
-		A.buffed++
-		A.maxHealth *= 1.5
-		A.health = min(A.maxHealth,A.health*1.5)
-		A.melee_damage_lower = max((A.melee_damage_lower * 2), 10)
-		A.melee_damage_upper = max((A.melee_damage_upper * 2), 10)
-		A.transform *= 2
-		A.environment_smash |= ENVIRONMENT_SMASH_STRUCTURES | ENVIRONMENT_SMASH_RWALLS
-		to_chat(user, "<span class='info'>You increase the size of [A], giving it a surge of strength!</span>")
-		qdel(src)
+	if(!proximity_flag || !isanimal(target))
+		return
+	var/mob/living/simple_animal/animal_target = target
+	if(animal_target.buffed || (animal_target.type in banned_mobs) || animal_target.stat)
```

Comment:
I don't see `.buffed` used anywhere else on simple animals.

Just remove the var entirely for simple animals, and just add a new trait `TRAIT_FUGU_WUMBO` or something like that.

Reply:
The concept of the variable is that if someone wanted to later down the line add a buff item for simplemobs akin to the fugu, they would also have it set buffed to true as to keep people from stacking buffs onto one mob.

Reply:
why, its a bad system. they can use literaly any other system.

Reply:
That variable has been there forever, no one's done anything with it.

Just remove the `.buffed` var on simple animals, and use a trait for tracking whether a fugu buff has happened. `TRAIT_FUGU_GLANDED`?

Reply:
I agree with coiax, make it a trait, if anyone cares downline then can modify the code to suit their needs